src/components/explore/ProbeExplorer.svelte

```diff
@@ -83,13 +83,29 @@
     QUARTER: 90,
   };
 
+  // Get the reference point from the query string if possible
+  function getDefaultReferencePoint() {
+    if ($store.ref) {
+      const found = data.find((d) => d[aggregationLevel] === $store.ref);
+      if (found) return found;
+    }
+    return data[data.length - 1];
+  }
+
+  let ref = getDefaultReferencePoint();
+
   function setDomain(str) {
-    if (aggregationLevel === 'build_id') {
+    if (str === 'ZOOM') {
```

Comment:
I would personally use constants for these things. e.g. `GLAM_ZOOM_KEY` or whatever